index.html

```diff
@@ -55,8 +55,12 @@
     <section class="py-5">
       <div class="container">
         <h1>Dr. Anita Sarma's research group at Oregon State University</h1>
-        <p class="lead">Investigating Human-Computer Interactions (HCI) across Software Engineering (SE) space.</p>
-        <p>This site is still under construction.</p>
+        <p class="lead">Our work follows an interdisciplinary approach of performing empirical user studies 
```

Comment:
Include an explanation of the EPIC acronym, which stands for _**E**nd-user centric, **P**rivacy, **I**nclusion and **C**ognition_. These are the primary research areas that our lab focuses on right now. Also, we don't actually do `"archival studies"`. The rest of this text is good, so try to weave it in with my suggestions if you can.